docs/conf.py

```diff
@@ -0,0 +1,21 @@
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.duration',
+    'sphinx.ext.autosectionlabel',
+]
+
+html_theme = 'alabaster'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'Envoy Gateway'
+copyright = '2022 Envoy Gateway Project Authors'
+author = 'Envoy Gateway Project Authors'
+
+version = '0.2.0-rc.1'
+release = '0.2.0-rc.1'
```

Comment:
is it possible to get these versions from somewhere more authoritative ?

Reply:
It needs to be made possible. 🙂

Reply:
Switched to `goversion` for this. 🙂